nuxt/index.js

```diff
@@ -52,21 +52,39 @@ module.exports = function nuxtBootstrapVue(moduleOptions = {}) {
       this.options.css.unshift('bootstrap/dist/css/bootstrap.css')
     }
 
-    // Transpile src/
+    // Component src prop resolving
+    this.options.build.loaders.vue.transformAssetUrls = {
+      // Ensure defaults are not lost
+      ...this.options.build.loaders.vue.transformAssetUrls,
```

Comment:
I think it would be better to put it last to prefer user-provided options.

Reply:
OK, I wasn't sure when user provided options were loaded... before or after this plugin is built

Reply:
Done